lib/oxidized/model/outputs.rb

```diff
@@ -1,5 +1,9 @@
+require 'oxidized/string'
+
 module Oxidized
   class Model
+    using Oxidized::StringRefinements
```

Comment:
We don't have to mention Oxidized, as we are also under this namespace, so the search path will hit us. Idiomatically we don't mention it, when we don't have to.

I personally still would prefer Oxidized::String to be module included in Refinements module which refines Strings by including the Oxidized::String module, as in the comware ticket.  But this is more matter of taste than anything I can justify. 

The justification being, maybe we add refinements to other classes, and then we don't have to change anything, as we already included the Refinements module, which contains not just String but now something else. 
And including module instead of listing it directly, because we can have them in separate files cleanly. 

Like maybe a refinements.rb including string and any futureType. 

And require 'refinements'. using Refinements.

Reply:
I like it, and I'll have a go at it, dunno if I can make it though :|

Reply:
> I personally still would prefer Oxidized::String to be module included in Refinements module which refines Strings by including the Oxidized::String module, as in the comware ticket. But this is more matter of taste than anything I can justify.

I don't think that's possibe. Or I don't know how.
I can create a Refinements Module and put the code there, so we only need that. But I don't think it's possible to "include" `Oxidized::String` in a `Refinements` module.

Reply:
we could do:

```ruby
module Refinements
  # Use the 'refine' keyword to define refinements for the String class
  refine String do
    [...]
  end

  refine Array do
    [...]
  end
end
```

```ruby
class Foobar
  using Refinements
  [...]
```

and that would allow us to refine String and Array at the same time.

Reply:
@ytti , so i tried to do come as close as possible to your wish. Please let me know if that's acceptable now :)